packages/clerk-js/src/utils/__tests__/ignoreEventValue.test.ts

```diff
@@ -1,7 +1,8 @@
-import { noop } from '@clerk/shared/testUtils';
-
 import { ignoreEventValue } from '../ignoreEventValue';
 
+// eslint-disable-next-line @typescript-eslint/no-empty-function
+const noop = (..._args: any[]): void => {};
```

Comment:
IMO fine to inline since this was only used once

Reply:
🔧 There are more references of noop in the codebase:
<img width="833" alt="Screenshot 2023-10-12 at 13 08 39" src="https://github.com/clerkinc/javascript/assets/3936408/1c88a689-296d-45bf-b519-13abe5ad10d0">

Reply:
`import { noop } from '@clerk/shared/testUtils` was only used once. The other imports won't be touched by the testUtils change